### PR TITLE
Tutorial on using TorchServe on Vertex AI

### LINKED ISSUE #2346: 💡 [REQUEST] - How to use TorchServe on Vertex
### 🚀 Descirbe the improvement or the new tutorial

Create a tutorial on how to use TorchServe on Vertex AI

### Existing tutorials on this topic

_No response_

### Additional context

_No response_

cc @msaroufim @agunapal @svekars @carljparker @NicolasHug @kit1980 @subramen

### PR DESCRIPTION
**Pull request by spzala**
This new tutorial is based on the Vertex AI documentation. The PR provides a similar doc as a tutorial hosted locally.

Fixes #2346 

## Description
This new tutorial is based on the Vertex AI documentation. The PR provides a similar doc as a tutorial hosted locally.
The link to the original documentation is provided at the end of the tutorial.

## Checklist

- [ x] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ x] Only one issue is addressed in this pull request
- [ x] Labels from the issue that this PR is fixing are added to this pull request
- [ x] No unnecessary issues are included into this pull request.

cc @msaroufim @agunapal @sekyondaMeta @svekars @carljparker @NicolasHug @kit1980 @subramen

### CODE REVIEW

**Comment by spzala** [comment]
cc @svekars - I have created this PR as per my comments https://github.com/pytorch/tutorials/issues/2346#issuecomment-1805084400 Thanks!

File: intermediate_source/torchserve_vertexai_tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Once you've uploaded the model artifacts into a GCS bucket, you can upload your PyTorch model to
+`Vertex AI Model Registry <https://cloud.google.com/vertex-ai/docs/model-registry/introduction>`__.
+From the Vertex AI Model Registry, you have an overview of your models
+so you can better organize, track, and train new versions. For this you can use the
+`Vertex AI SDK <https://cloud.google.com/vertex-ai/docs/python-sdk/use-vertex-ai-python-sdk>`__
+and this 
+`pre-built PyTorch container <https://cloud.google.com/blog/products/ai-machine-learning/prebuilt-containers-with-pytorch-and-vertex-ai>`__.
+
+
+.. code:: shell
```
Comment (agunapal) [inline intermediate_source/torchserve_vertexai_tutorial.rst:77]: This section on uploading model doesn't have the correct code. Can you please check the notebook and edit this

File: intermediate_source/torchserve_vertexai_tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        accelerator_type="NVIDIA_TESLA_P100",
+        accelerator_count=1,
+        traffic_percentage=100,
+        deploy_request_timeout=1200,
+        sync=True,
+    )
+
+If you follow the
+`notebook <https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/vertex_endpoints/torchserve/dreambooth_stablediffusion.ipynb>`__
+you can also get online predictions using the Vertex AI SDK.
```
Comment (agunapal) [inline intermediate_source/torchserve_vertexai_tutorial.rst:111]: Including the prediction API here with the output would make the example complete

**Review by agunapal — CHANGES_REQUESTED** [review]

File: intermediate_source/torchserve_vertexai_tutorial.rst (hunk 1 above)
Comment (spzala) [reply intermediate_source/torchserve_vertexai_tutorial.rst:77]: @agunapal thank you so much for the review! Good catch! I will update it.

**Review by agunapal — APPROVED** [review]
LGTM

**Comment by spzala** [comment]
Thank you so much @svekars !!
